conceptual/Aspects/advising/sharing-state.md

```diff
@@ -17,7 +17,7 @@ The inconvenience of this method is that it is not very flexible. You cannot pas
 
 ## Sharing state with the Tags property
 
-If your `BuildAspect` method needs to pass to a template method some state that is specific to an advice instance, you can construct an instance of the <xref:System.Collections.Generic.Dictionary{System.String,System.Object}> class and assign all elements of state as tags_ in name-value dictionary. In the template method, the tags are available under the `meta.Tags` dictionary.
+If your `BuildAspect` method needs to pass to a template method some state that is specific to an advice instance, you can construct an instance of the <xref:System.Collections.Generic.Dictionary`2><<xref:System.String>, <xref:System.Object>> class and assign all elements of state as tags in the name-value dictionary. In the template method, the tags are available under the `meta.Tags` dictionary.
```

Comment:
`<xref:System.Collections.Generic.Dictionary`2><<xref:System.String>, <xref:System.Object>>` does not seem correct. Did you try to compile?